cmd/internal/flowmode/cmd_run.go

```diff
@@ -99,19 +101,23 @@ depending on the nature of the reload error.
 		StringVar(&r.clusterJoinAddr, "cluster.join-addresses", r.clusterJoinAddr, "Comma-separated list of addresses to join the cluster at")
 	cmd.Flags().
 		BoolVar(&r.disableReporting, "disable-reporting", r.disableReporting, "Disable reporting of enabled components to Grafana.")
+	cmd.Flags().StringVar(&r.converterSourceFormat, "converter.source-format", r.converterSourceFormat, "The format of the source file. Supported formats: 'prometheus'.")
+	cmd.Flags().BoolVar(&r.converterBypassWarnings, "converter.bypass-warnings", r.converterBypassWarnings, "Enable bypassing warnings when converting")
```

Comment:
What do you think about putting these flags in the `config` namespace to make it more clear what's happening, e.g., `--config.format` and `--config.bypass-conversion-warnings`? 

`converter` makes it sound like there's some converter process always running, when really this is just processing done on the config file. 

(When I was playing around with the `convert` subcommand, I found `--source-format` to be weirdly long; `--format` might make enough sense since the target format is always Flow)

Reply:
done